### ISSUE TITLE
Create docktie_cli

### ISSUE THREAD

**Issue by casimpania** [docktie/docktie#2]
**docktie_cli** - envisioned as the only command a user needs to remember. It should help out in every possible tool usage.

**Initial subcommands idea:**
* help - all available commands. Default when no commands given. 
* list - lists all utilities (within utils)
* shell <service_name> - ssh to service name
* services - list all the docker services

Do after #1 is done

**Comment by icasimpan** [comment]
Deferred full implementation to v0.2.x

**Comment by icasimpan** [comment]
Closing as this will be continued in https://github.com/docktie/docktie/issues/16
